packages/stream_chat/lib/src/client/channel.dart

```diff
@@ -258,8 +258,34 @@ class Channel {
     return state!.channelStateStream.map((cs) => cs.channel?.cooldown ?? 0);
   }
 
+  /// Remaining cooldown duration in seconds for the channel.
+  ///
+  /// Returns 0 if there is no cooldown active.
+  int get remainingCooldown {
```

Comment:
I have the feeling this is quite a heavy getter. Usually when you call a get you expect a simple value.
Maybe we could change it to a method and rename to `getRemainingCooldown` or `calculateRemainingCooldown`? So devs are aware it is not recommended to call it in the build methods.

Reply:
Renamed to `getRemainingCooldown()`